conan/tools/gnu/pkgconfigdeps/pkgconfigdeps.py

```diff
@@ -11,8 +11,13 @@ def __init__(self, conanfile):
     def content(self):
         """Get all the *.pc files content"""
         pc_files = {}
+        # Get all the dependencies
         host_req = self._conanfile.dependencies.host
-        for _, dep in host_req.items():
+        build_req = self._conanfile.dependencies.direct_build
+        test_req = self._conanfile.dependencies.test
+        for _, dep in list(host_req.items()) + list(build_req.items()) + list(test_req.items()):
```

Comment:
Not build_requires, only ``test_requires`` should generate .pc files

Build_requires are things like "cmake"

Reply:
Ok. I saw CMakeDeps doing that so I only replicated that behavior